packages/web-app-draw-io/README.md

```diff
@@ -0,0 +1,3 @@
+# web-app-draw-io
+
+This application can be used for creating, opening and editing `.draw` files.
```

Comment:
It also lets you import some visio file type... not sure how good that works though.